.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [10.x]
+        node-version: [14.x]
```

Comment:
Can this be updated to `v16.20.2`. I will have a PR soon that moves us to this node version.

Reply:
not without further changes. I haven't look to deeply into fixing tests for Node 16. I wanted to keep the changes at a minimum

Reply:
Probably not an issue for workflows hoever.